README.md

```diff
@@ -7,61 +7,198 @@
+
+Are you begining to see the pattern?  Let's look at another one, maybe something more like an actual program, like "if the file exists then open it, otherwise tell the user it can't be found".  Let's see what it would look like in code:
+
+````Swift
+let doesFileExist = findFileNamed("testfile")
+if doesFileExist == true {
+	openFileNamed("testfile")
 } else {
-    print("I'm sorry, you're too short to go on this ride today.")
+	reportError("testfile not found")
 }
+````
+
+Every Conditional effectively has both an ````if```` and an ````else```` part but depending upon how we style our code we can leave out the explicit ````else```` (which the compiler interprets as "if the condition is false then don't do anything").  For example, we could re-write the code above as follows:
 
-// prints "I'm sorry, you're too short to go on this ride today."
-```
+````Swift
+let doesFileExist = findFileNamed("testfile")
+if doesFileExist == false {
+	reportError("testfile not found")
+	return
+}
+openFileNamed("testfile")
+````
 
-* Create/chain multiple *if* statements together to consider additional clauses:
+Can you see how they'e variations of the same thing?  Let's look at another example.
 
-```swift
-var temperatureInFahrenheit = 100
+````Swift
+var rainingOutside = false
 
-if temperatureInFahrenheit < 32 {
-    print("Snow! 🌨")
-} else if temperatureInFahrenheit >= 90 {
-    print("Very hot! ☀️")
+if rainingOutside == true {
+	takeUmbrella()
 } else {
-    print("Just right. 🌻")
+	// don't worry about taking the umbrella
+	// NB no code is being excuted here, only comments!
+}
+goForAWalk()
+````
+is the same as...
+
+````Swift
+func takeUmbrella() {
+    // grab umbrella
+    print("I have my umbrella! ☂️")
 }
-```
 
 
+}
+goForAWalk()
+````
 
-## What the student can do at this point 
+Our ````if```` doesn't have to be followed by a Boolean expression; it can also take Boolean variable or constant, or a function which returns a Boolean.  Here's what that would look like in code:
 
-* Create variables and constants
-* Is familiar with type annotations, type inference and string interpolation.
-* Can create functions with return types.
-* Is familiar with the `String`, `Int`, `Double` and `Bool` type.
-* Just learned about `Bool` prior to this reading.
-* Just learned about *comparison operators* but has YET to see what an if statement looks like. 
+````Swift
+let x = true
+if x {
+	...
+}
+````
 
+is the same as
 
-## Outline / Notes
+````Swift
+let x = true
+if x == true {
+	...
+}
+````
+
+And we can use variables:
+
+````Swift
+var y = false
+if y {
+...
+}
+````
+
+Which is the same as:
+
+````Swift
+var y = false
+if y == true {
+   ...
+ }
+ ````
+ 
+And here we'll show how to use a function:
+
+````Swift
+func test(Bool a, Bool b) -> Bool {
+...
+}
+
+if test(x, y) {
+...
+}
+````
```

Comment:
The format of the `test` function won't compile here. Here is how this should read here:

---
```swift
func test(a: Bool, b: Bool) -> Bool {
    ...
}


if test(x, b: y) {
    ...
}
```